### ISSUE TITLE
Tunnel.Consumer is empty when using @stencil/router >0.2.6

### ISSUE THREAD

**Issue by MattMcFarland** [ionic-team/stencil-state-tunnel#6]
I finally figured out what was causing my problems in #5 
it looks like stencil router is pulling in the consumer as well, but your docs are using ion-router which appears to work.

Should we be using ion router instead?

**Comment by MattMcFarland** [comment]
@joelalejandro it looks like youre using an old version of router in https://github.com/counterfactual/monorepo/blob/master/packages/dapp-high-roller/package.json (0.2.6) 

Youre also not pulling in state-tunnel as a dependency, and therefore using the state-tunnel through the one that router is getting.

So looking at your yarn.lock:
https://github.com/counterfactual/monorepo/blob/master/yarn.lock#L755-L784

Maybe one of your other apps is pulling in the other version of state-tunnel - but you get 0.0.91 from the router, and when i use `yarn add @stencil/state-tunnel` it gives me 0.0.8 - and if I have both state-tunnel and router as dependencies, typescript blows up saying that the consumer component has already been defined.

I'm really at a loss atm, I can't get state-tunnel consumer working at all.  I can if I rebuild an app from scratch, but whats really throwing me off right now is that the dependency versions all over the place, like maybe I have to get my versions all exact for it to work.  I did not have any issues at all until I updated stencil's core.  Are they even doing integration testing between their components? I swear something is going on

**Comment by joelalejandro** [comment]
Oh right, I do remember we are forced to use `@stencil/router@0.2.6`. We got it working with the following versions:

@stencil/core: 0.15.2 (or 0.16.0)
@stencil/router: 0.2.6
@stencil/state-tunnel: 0.0.8

And yeah, I know what you mean. Stencil works great with Web Components, but it looks like beyond that, the developer experience is a bit clunky yet.

**Comment by MattMcFarland** [comment]
Your suggestion fixed my issue.  We probably want to leave this open as stencil should-state-tunnel should probably work with the latest version of router and core. I'm going to rename this topic.

**Comment by joelalejandro** [comment]
Glad to hear @MattMcFarland! Yeah I agree, let's keep the issue open :)

**Comment by John0x** [comment]
Any progress on this?
Still not working with the latest version

**Comment by jthoms1** [comment]
Working on it now. Hopefully I will have a fix out this week.

**Comment by simonhaenisch** [comment]
If by empty you mean that it renders an empty `<tunnel-consumer></tunnel-consumer>`, then we have the same problem here, but only on Windows. My work mate was just telling me about some components not rendering, and I couldn't reproduce on Mac, so I got her to share her screen and saw that empty tunnel consumer. Tried a couple things, like upgrading to `0.0.9-5` but didn't help. Wondering whether all those Stencil fixes related to paths on Windows are going to help... will try to migrate our code base next week.
